src/main/java/service/LottoService.java

```diff
@@ -0,0 +1,68 @@
+package service;
+
+import model.*;
+import utils.enums.ConstantNumber;
+import utils.generator.NumbersGenerator;
+
+import java.util.*;
+
+public class LottoService {
+    private final NumbersGenerator numbersGenerator;
+
+    public LottoService(NumbersGenerator numbersGenerator) {
+        this.numbersGenerator = numbersGenerator;
+    }
+
+    public Lottos generateLottos(int price) {
+        int numberOfLottos = price / ConstantNumber.LOTTO_PRICE_UNIT.getNumber();
+        return new Lottos(generateLottoList(numberOfLottos));
+    }
+
+    private List<Lotto> generateLottoList(int numberOfLottos) {
+        List<Lotto> lottoList = new ArrayList<>();
+        for (int i = 0; i < numberOfLottos; i++) {
+            List<Integer> numbers = new ArrayList<>(numbersGenerator.generate());
+            Collections.sort(numbers);
+            lottoList.add(new Lotto(numbers));
+        }
+        return lottoList;
+    }
+
+    public LottoResult calculateLottoResult(Lottos lottos, WinningLotto winningLotto) {
+        Map<LottoRank, Integer> lottoRanks = new HashMap<>();
+        for (LottoRank rank : LottoRank.values()) {
+            lottoRanks.put(rank, 0);
+        }
+        for (Lotto lotto : lottos.getLottoList()) {
+            LottoRank rank = calculateRank(lotto, winningLotto);
+            lottoRanks.put(rank, lottoRanks.get(rank) + 1);
+        }
```

Comment:
```suggestion
        Map<LottoRank, Integer> lottoRanks = new HashMap<>();
        for (Lotto lotto : lottos.getLottoList()) {
            LottoRank rank = calculateRank(lotto, winningLotto);
            lottoRanks.put(rank, lottoRanks.getOrDefault(rank, 0) + 1);
        }
```

이렇게 바꾸면 코드도 짧아지고 맵에 0값이 밸류인 요소도 없어져서 좋지 않을까 생각합니다!

Reply:
훨씬 깔끔하네요!